FOSS/themes/hugo-theme-learn/layouts/partials/header.html

```diff
@@ -135,7 +135,7 @@ <h1>
           {{ if $parent }}
           {{ template "breadcrumb" (dict "page" $parent "value" "" "count" (add .count 1)) }}
           {{ end }}
-          {{ $value := (printf "%s<a itemprop='itemListElement' itemtype='http://schema.org/ListItem' href='%s'>%s<meta itemprop='name' content='%s'><meta itemprop='position' content='%d'></a>" .value .page.RelPermalink .page.Title .page.Title .count) }}
+          {{ $value := (printf "%s<span class='links' itemprop='itemListElement' itemscope itemtype='https://schema.org/ListItem'><a class='highlight' itemprop='item' href='%s'>%s<meta itemprop='name' content='%s'/></a><meta itemprop='position' content='%d'></span>" .value .page.RelPermalink .page.Title .page.Title .count) }}
```

Comment:
```suggestion
          {{ $value := (printf "%s<span class='links' itemprop='itemListElement' itemscope itemtype='http://schema.org/ListItem'><a class='highlight' itemprop='item' href='%s'>%s<meta itemprop='name' content='%s'/></a><meta itemprop='position' content='%d'></span>" .value .page.RelPermalink .page.Title .page.Title .count) }}
```

Google insists on http